acquia-http-hmac.gemspec

```diff
@@ -13,6 +13,9 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.executables << 'acq-http-request'
 
+  s.add_dependency('commander', '~> 4.4')
```

Comment:
This gem has intentionally avoided introducing dependencies to avoid code bloat in projects which depend on it.  This will cause these gems to be required for every ruby package which depends on http-hmac-ruby, regardless of whether the CLI client is actually used.  Can we make these development dependencies instead?